### ISSUE TITLE
Doc: Getting started, set up of dev environment.

### ISSUE THREAD

**Issue by haru-02** [debian-cryptoanarchy/cryptoanarchy-deb-repo-builder#218]
Hello everyone,

New developer here looking to set up the dev environment to start contributing to the project. But i did not find the development documentation adequate to get started as a newcomer. I am planning to contribute and improve the "getting started" documentation for this project for now. So any help on how to get the dev env set up to code would be nice. I am using Linux mint 21. should i have debian 10 installed to contribute to this project ? or can i just run this on a VM ?

Thank you.

**Comment by Kixunil** [comment]
Yeah, the state of dev doc is poor. :(

You can use VM just fine, in fact that's the primary development tool for me.
Please look at the CI script for now, it contains all the commands needed to set up the machine for building. You don't need to use additional container if you already have Debian 10 VM though.

(Note also that CI is broken, I have a solution but it's a bit involved, so I didn't get to implementing it.)

**Comment by haru-02** [comment]
thank you! i am checking out the CI script right now. Also, i only have 8gb of ram on my laptop. That should be fine for development right ?

**Comment by haru-02** [comment]
https://hollowmansblog.wordpress.com/2022/08/13/my-summer-of-bitcoin-2022-project-ci-for-cadr/

Also, is this the CI being referred to ? Thanks for clarifying :)

**Comment by Kixunil** [comment]
Yes, my VMs have even less RAM. Lot of RAM is only required if you're trying to sync mainnet *fast* but 8G should be sufficient even for that.
